src/accuracy_checker/accuracy_checker.py

```diff
@@ -1,59 +1,78 @@
-import os
-import sys
 import argparse
 import logging as log
-from parameters import parameters
-from executors import executor
-from process import process
-from output import output_handler as out_hand
-from config_parser import parser
+import os
+import sys
 
+from config_parser import Parser
+from executors import Executor
+from output import OutputHandler
+from parameters import Parameters
+from process import Process
+
+
+def cli_parser():
```

Comment:
please set more clear name like "cli_argument_parser".
In this case docstring will not be needed